src/vis/Vis.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Vis } from './Vis';
+import { EColumnTypes, VisColumn } from './interfaces';
+
+function fetchData(numberOfPoints: number): VisColumn[] {
+  const dataGetter = async () => {
+    await new Promise((resolve) => {
+      setTimeout(resolve, 1500);
+    });
+
+    return {
+      value: Array(numberOfPoints)
+        .fill(null)
+        .map(() => Math.random() * 100),
+      pca_x: Array(numberOfPoints)
+        .fill(null)
+        .map(() => Math.random() * 100),
+      pca_y: Array(numberOfPoints)
+        .fill(null)
+        .map(() => Math.random() * 100),
+      category: Array(numberOfPoints)
+        .fill(null)
+        .map(() => parseInt((Math.random() * 10).toString(), 10).toString()),
+    };
+  };
+
+  const dataPromise = dataGetter();
+
+  return [
+    {
+      info: {
+        description: '',
+        id: 'pca_x',
+        name: 'pca_x',
+      },
+      type: EColumnTypes.NUMERICAL,
+      values: () => dataPromise.then((data) => data.pca_x.map((val, i) => ({ id: i.toString(), val }))),
+    },
+    {
+      info: {
+        description: '',
+        id: 'pca_y',
+        name: 'pca_y',
+      },
+      type: EColumnTypes.NUMERICAL,
+      values: () => dataPromise.then((data) => data.pca_y.map((val, i) => ({ id: i.toString(), val }))),
+    },
+    {
+      info: {
+        description: '',
+        id: 'value',
+        name: 'value',
+      },
+      type: EColumnTypes.NUMERICAL,
+      values: () => dataPromise.then((data) => data.value.map((val, i) => ({ id: i.toString(), val }))),
+    },
+    {
+      info: {
+        description: '',
+        id: 'category',
+        name: 'category',
+      },
+      type: EColumnTypes.CATEGORICAL,
+      values: () => dataPromise.then((data) => data.category.map((val, i) => ({ id: i.toString(), val }))),
+    },
```

Comment:
Should we add some proper names and description?

Reply:
I just needed some random demo data to show something, but maybe we should show some data that makes sense? I am thinking about something like this [iris ](https://www.google.com/search?q=iris+dataset&oq=iris+dataset&aqs=chrome.0.0i512l10.1749j0j7&sourceid=chrome&ie=UTF-8)dataset? Do you have any dataset in mind where we can visually present all use-cases with a single dataset @thinkh  @dvzacharycutler?